carlson_cummings_hisada_lab01.Rmd

```diff
@@ -156,10 +160,6 @@ count.by(PctEmployed16_Over, is.na)
 cleanPctEmployed16_Over <- !is.na(PctEmployed16_Over)
 ```
 
-\medskip
-\medskip
-\medskip
-
```

Comment:
I removed these medskips because one line would show on its own page, this brought that one line back up to the existing page.